nixos/modules/system/boot/kernel.nix

```diff
@@ -77,8 +77,8 @@ in
       type = types.int;
       default = 4;
       description = ''
-        The kernel console log level.  Log messages with a priority
-        numerically less than this will not appear on the console.
+        The kernel console `loglevel`. All Kernel Messages with a log level smaller
```

Comment:
```
`loglevel`
```

should probably be `<literal>loglevel</literal>`

Reply:
fixed